sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -387,24 +397,24 @@ public void addEvent(io.opentelemetry.trace.Event event, long timestamp) {
     addTimedEvent(TimedEvent.create(timestamp, event));
   }
 
-  private Map<String, AttributeValue> limitEventAttributes(Map<String, AttributeValue> attributes) {
-    if (attributes.size() <= this.maxNumberOfAttributesPerEvent) {
-      return attributes;
+  static Map<String, AttributeValue> copyAndLimitAttributes(
```

Comment:
Since this method is also used elsewhere, I'd rather see it moved to a separate utility class, rather than leave it in here.

Reply:
Can I do it in a separate PR?

Reply:
yessir!